### PR TITLE
Added PickPhotosAsync

### PR DESCRIPTION
**Pull request by GUOLDEV**
### Description of Change ###

Implementing PickPhotosAsync to MediaPicker

### Bugs Fixed ###

- Related to issue #

Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.

### API Changes ###

List all API changes here (or just put None), example:

Added: 
 
- `string Class.Property { get; set; }`
- `void Class.Method();`

Changed:

 - `object Cell.OldPropertyName` => `object Cell.NewPropertyName`
 
If there is an entirely new API, then you can use a more verbose style:

```csharp
public static class NewClass {
    public static int SomeProperty { get; set; }
    public static void SomeMethod(string value);
}
```

### Behavioral Changes ###

Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of `main` at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

**Comment by dimonovdd** [comment]
I didn't see the implementation in this PR. I also think that we cannot make such PRs until #1475 is merged

File: Xamarin.Essentials.sln [hunk 1]
```diff
@@ -19,12 +19,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tests", "Tests\Tests.csproj
 EndProject
```
Comment (dimonovdd) [inline Xamarin.Essentials.sln:19]: You should probably discard the change to this file

Comment (GUOLDEV) [reply Xamarin.Essentials.sln:19]: I will remove it, thanks 👍

**Comment by GUOLDEV** [comment]
> I didn't see the implementation in this PR. I also think that we cannot make such PRs until #1475 is merged

Hi @dimonovdd , I just started trying to find the best way to get this task done, created this draft PR for visualisation and to get help to achieve the implementation. Please let me know if you have any questions or ideas how we can get it done 👍

**Comment by dimonovdd** [comment]
You can copy the implementation from [here](https://github.com/dimonovdd/Xamarin.MediaGallery). 
But before you do this we need to combine PR #1475

**Comment by GUOLDEV** [comment]
Hi @dimonovdd, I did a quick look at your plugin and it looks great! It is definitely going to be a good addition to this plugin.

**Comment by dimonovdd** [comment]
I think this will not become part of Xamarin.Essentials. But I will try to add this to MAUI.Essentials

**Comment by GUOLDEV** [comment]
@dimonovdd I worked a little bit on this issue and I got it working for iOS, I am having issues getting the correct file path using PHPickerResult. I will try to get it working for Android as well.

**Comment by jfversluis** [comment]
This won't be happening for Xamarin.Essentials anymore unfortunately. Thanks so much for your time and effort here! We will make this happen for .NET MAUI!

**Pull request closed without merging**